src/ForeignExchangeReferenceRates.php

```diff
@@ -16,19 +16,19 @@ public function getProviderName()
 
     public function fetch(Connector $connector)
     {
-        $xml = $connector->fetch(self::URL);
-        $crawler = new Crawler($xml);
+        $xmlString = $connector->fetch(self::URL);
+        $xml = simplexml_load_string($xmlString);
 
-        $ratesContainer = $crawler->filterXPath('./*/default:Cube/default:Cube[1]');
-        $date = $ratesContainer->attr('time');
+        $ratesContainer = $xml->Cube->Cube;
+        $date = (string) $ratesContainer['time'];
```

Comment:
Please eliminate the space after the cast, consistent with the casts on lines 30-31.